sceptre/strides/config/prod/github-oidc-sage-bionetworks-synapse-login.yaml

```diff
@@ -0,0 +1,17 @@
+template:
+  type: http
+  url: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/IAM/github-oidc-provider.j2
+stack_name: github-oidc-sage-bionetworks
```

Comment:
name should probably contain matching repo name in it, `synapse-login-aws-infra`